### PR TITLE
Add Firmware 2.0.4.6.41

### PR DESCRIPTION
**Pull request by fouille**
For Devices : UC926 UC924 UC923 UC921 UC912G UC903 UC902

Jira-Issue: https://wazo-dev.atlassian.net/browse/WAZO-1750

### CODE REVIEW

File: plugins/wazo-htek/2.0.4.6.41/entry.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (C) 2017 Wazo Authors
```
Comment (fblackburn1) [inline plugins/wazo-htek/2.0.4.6.41/entry.py:3]: 2017 !? You live in the past?

Comment (fouille) [reply plugins/wazo-htek/2.0.4.6.41/entry.py]: :D corrected

**Comment by bloom1** [comment]
@fouille has this been tested on a phone?

**Comment by sboily** [comment]
Can we merge it in testing repo for people would like to test easily with real phone? Except if you think there is issue, but looks like good for me.

File: plugins/wazo-htek/build.py [hunk 2]
```diff
@@ -20,11 +20,18 @@
 
 from subprocess import check_call
 
-
```
Comment (fblackburn1) [inline plugins/wazo-htek/build.py:23]: :scream: https://www.python.org/dev/peps/pep-0008/#blank-lines

Comment (fblackburn1) [reply plugins/wazo-htek/build.py]: you're not funny @bloom1 :stuck_out_tongue_closed_eyes:

**Comment by bloom1** [comment]
> Can we merge it in testing repo for people would like to test easily with real phone? Except if you think there is issue, but looks like good for me.

Done. It is also possible to test a plugin without merging it into testing: Zuul builds the plugins for every PR. You can see here: https://zuul.wazo.community/logs/45/45/2e0016a210097905f11bf9e553fc08888b799a66/check/wazo-provd-plugins/9a35292/plugins/
It is possible to use this URL in the provisioning configuration :)

**Comment by sboily** [comment]
> > Can we merge it in testing repo for people would like to test easily with real phone? Except if you think there is issue, but looks like good for me.
> 
> Done. It is also possible to test a plugin without merging it into testing: Zuul builds the plugins for every PR. You can see here: https://zuul.wazo.community/logs/45/45/2e0016a210097905f11bf9e553fc08888b799a66/check/wazo-provd-plugins/9a35292/plugins/
> It is possible to use this URL in the provisioning configuration 😃

well for our community users it's quite complicated...

**Pull request merged**